com.microsoft.mrtk.input/Interactors/InteractorVisuals/MRTKRayReticleVisual.cs

```diff
@@ -112,18 +112,23 @@ private void UpdateReticle()
                     }
                     else
                     {
-                        Reticle.SetActive(false);
-                    }
-
-                    // The proximity light should only be active when the reticle is
-                    if (proximityLight != null)
-                    {
-                        proximityLight.SetActive(Reticle.activeSelf);
+                        ReticleSetActive(false);
                     }
                 }
             }
         }
 
+        private void ReticleSetActive(bool value)
```

Comment:
Would something like `ReticleAndProxLightSetActive` or `SetActiveReticleAndProxLight` makes more sense?

Reply:
prox lights might not always be present so I wanted to focus solely on the reticle aspect with the naming

Reply:
I understand the point, and also the name of a private function does not matter that much. It's just that I was confused when I saw the change from `Reticle.SetActive(true);` to `ReticleSetActive(true);` before reading the content of the function. I am OK either way.